scene/resources/default_theme/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Don't ignore some graphics. Overrides Godot's global gitignore (specifically the part that filters out code coverage.)
+!*.png
```

Comment:
Could be solved in the main `.gitignore` by making the code coverage ignores absolute.

CC @qarmin, are those always generated in the root folder?

Reply:
Not always, but depends where user execute specific commands. 
I think that only sensible directories in which user may execute generation of html files are root(/) or bin folder(which is hidden by default in git.)

So changing this records to point to absolute path are fine to me
```
MachineIndependent/
godot.info
amber.png
emerald.png
glass.png
ruby.png
snow.png
updown.png
gcov.css
```

Reply:
Would those files show up in the actual root directory, or in a subdirectory? Alternatively, can you give me the magic command needed to generate that coverage report so I can test it? :)

Reply:
For now I cannot test how exactly it works but this are steps how to get coverage report(works only on Linux and 4.0 branch):
```
sudo apt install gcc gcov lcov clang # Some may be not available
scons p=linuxbsd use_coverage=yes
bin/godot.linuxbsd.tools.64s
lcov --directory ./ --capture --output-file godot.info
genhtml godot.info --ignore-errors source
lcov --zerocounters # To clear results
xdg-open index.html # opens results
```

Reply:
Unfortunately it looks like this doesn't function under VirtualBox, which means I can't test it either.

However, looking at the commandline, it looks like you can specify a directory for this stuff to end up in. Maybe we should just ignore an `lcov` or `coverage` directory, then tell people to put their coverage data in there instead of in the root directory?